frontend/src/metabase/services.js

```diff
@@ -103,10 +103,10 @@ export const CardApi = {
   delete: DELETE("/api/card/:cardId"),
   query: POST("/api/card/:cardId/query"),
   query_pivot: POST("/api/card/pivot/:cardId/query"),
-  // isfavorite:                  GET("/api/card/:cardId/favorite"),
-  favorite: POST("/api/card/:cardId/favorite"),
-  unfavorite: DELETE("/api/card/:cardId/favorite"),
-
+  bookmark: {
+    create: POST("/api/card/:id/bookmark"),
```

Comment:
This is not the correct endpoint. 
To create a card bookmark we'll need to use POST("/api/bookmark/card/:id")

Likewise with DELETE

Reply:
Thank, will change.